File: C-Abi-Bridge-Aot/Src/CAbiBridge.Compress/NetServicesCompress/NetServiceFileCompressPackage/Enums/FcpType.cs


namespace michele.natale.Compresses;


/// <summary>
/// Defines the type of FileCompressPackage (FCP) 
/// entry stored in the archive.
/// </summary>
public enum FcpType : byte
{
  /// <summary>
  /// Represents a single file entry.  
  /// The file data is stored in compressed form.
  /// </summary>
  File = 1,

  /// <summary>
  /// Represents a multi-file archive entry.  
  /// The archive contains multiple files and directories packed together.
  /// </summary>
  Archiv = 2,
}

File: C-Abi-Bridge-Aot/Tests/CAbiBridge.Aot.Vb.Tests/TestCompress/TestFileCompressPackage/Enum/FcpType.vb
Option Strict On
Option Explicit On

Namespace michele.natale.Tests


  ''' <summary>
  ''' Defines the type of FileCompressPackage (FCP) 
  ''' entry stored in the archive.
  ''' </summary>
  Friend Enum FcpType As Byte
    ''' <summary>
    ''' Represents a single file entry.  
    ''' The file data is stored in compressed form.
    ''' </summary>
    File = 1

    ''' <summary>
    ''' Represents a multi-file archive entry.  
    ''' The archive contains multiple files and directories packed together.
    ''' </summary>
    Archiv = 2
  End Enum
End Namespace